modules/@apostrophecms/blog-page/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+
+  options: {
+    label: 'aposBlog:page',
+    piecesFilters: [{ name: 'year' }, { name: 'month' }, { name: 'day' }]
```

Comment:
Add our standard eslint package settings to this module and make sure it complies.

Reply:
Done